db/people.yml

```diff
@@ -952,7 +952,11 @@ rpokratath:
   aka:
     - Rohan Pokratath
   avatar: https://www.github.com/rohanpok.png
-  bio: bio
+  bio: "Rohan was born in Calicut, India in 1993. He did his masters in Material Science\
+    \ from Indian Institute of Technology Bombay and currently enrolled as a PhD student\
+    \ at University of Basel, on the topic of characterizing metal oxide nanocrystals\
+    \ using X-ray Pair Distribution Function Analysis.  In his spare time, he enjoys football."
+
```

Comment:
- please remove empty line here
 - please remove end_month and end_year from your PhD entry in education (you haven't finished yet!)
 - please could you add a sentence in the bio which explains your Columbia visit.  This may not be a big thing overall but if and when this builds into our group web-page it will help people understand why you are here.